pages/blog/[slug].tsx

```diff
@@ -45,6 +45,7 @@ export default function Post({ post, morePosts, preview, slug }: Post) {
   const postUrl = post?.slug
     ? `https://kedro.org/blog/${post.slug}`
     : 'https://kedro.org';
+  const mastodonContent = 'See more: https://social.lfx.dev/@kedro';
```

Comment:
You could move this outside of the `<Post />` component since it doesn't depend on any state/events inside it.

Reply:
Good call! I'll likely move it to the bottom of the slug file and outside of the Post component function - otherwise perhaps the `utils` file? Not sure where's the best placement for it

Reply:
Fairs fairs, sounds good!